misk-hibernate/src/main/kotlin/misk/hibernate/HibernateExceptionMappers.kt

```diff
@@ -14,29 +14,35 @@ import org.slf4j.event.Level
 import jakarta.inject.Inject
 import javax.persistence.OptimisticLockException
 
-internal class RetryTransactionExceptionMapper @Inject internal constructor() :
+internal class RetryTransactionExceptionMapper @Inject internal constructor(
+  val config: HibernateExceptionLogLevelConfig? = null,
```

Comment:
I added default value to these injected dependencies to avoid `revapi` breaking change, but they seem redundant to me. I'm wondering if it makes more sense to make as an API breaking change instead?